tutorials/deploy-mean-app-mongodb-replication.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Deploy a Production-Ready MEAN Application using Bitnami MongoDB with Replication on Google Cloud Platform
+description: Deploy a MEAN application with a multi-node MongoDB replica set on Google Cloud Platform using Bitnami MEAN and Bitnami MongoDB with Replication.
+author: vikram-bitnami
+tags: mean, mongodb, replication, scalability, Google Cloud Platform, bitnami
+date_published: 2017-07-06
+---
+
+This tutorial demonstrates how to deploy a MEAN (MongoDB, Express, Angular and Node) Web application with a multi-node MongoDB replica set on Google Cloud Platform using Bitnami MEAN and Bitnami MongoDB with Replication. This approach produces a more scalable and resilient deployment that could be used for production or mission-critical scenarios. This tutorial assumes that you're familiar with MongoDB and MEAN applications.
+
+## Objectives
+
+* Deploy Bitnami MEAN on a Google Compute Engine instance.
+* Install and configure a sample MEAN application.
+* Serve the application with Apache.
+* Deploy Bitnami MongoDB with Replication on a set of Google Compute Engine instances.
+* Connect the MEAN application to the Bitnami MongoDB with Replication cluster.
+* Test service continuity (optional)
+
+## Before you begin
+
+Before starting this tutorial, ensure that you have set up a Google Cloud Platform project. You can use an existing project or [create a new project](https://console.cloud.google.com/project).
+
+## Cost
+
+The default configuration allows you to run a MEAN application using 1 `f1-micro` instance with a standard 10 GB persistent disk for the application server, and 3 `n1-standard-2` instances, each with a standard 10 GB persistent disk, for the MongoDB cluster. You can customize the configuration when deploying this solution or change it later, although the default configuration is fine for the purposes of this tutorial.
+
+Estimated cost for the above default configuration is $4.28 per month for the single `f1-micro` instance and $148.04 per month for the three `n1-standard-2` instances comprising the Bitnami MongoDB with Replication cluster, based on 30-day, 24 hours per day usage in the Central US region. Sustained use discount is included.
+
+Use the [pricing calculator](https://cloud.google.com/products/calculator/) to generate a cost estimate based on your projected usage. New Google Cloud customers may be eligible for a [free trial](https://cloud.google.com/free-trial).
+
+## Deploy Bitnami MEAN on a Google Compute Engine instance
+
+Deploy Bitnami MEAN on a Google Compute Engine instance:
+
+1. From the [Cloud Launcher](https://console.cloud.google.com/launcher), select the [`MEAN Certified by Bitnami`](https://console.cloud.google.com/launcher/details/bitnami-launchpad/meanstack) template.
```

Comment:
Change "https://console.cloud.google.com/launcher" to "https://console.cloud.google.com/marketplace" here and throughout